application/config/config-defaults.php

```diff
@@ -646,6 +646,20 @@
 // Use alias notation, we should move to this format everywhere.
 $config['plugindir']               = 'webroot.plugins';
 
+// AuditLog Default values
+$config['auditLog'] = array(
```

Comment:
auditlog is a plugin : please don't add system for specific plugin in core.

Each plugin setting allow a default value too